langchain/src/chat_models/baiduwenxin.ts

```diff
@@ -0,0 +1,425 @@
+import axiosMod, { AxiosResponse, AxiosStatic } from "axios";
+import { BaseChatModel, BaseChatModelParams } from "./base.js";
+import {
+  AIChatMessage,
+  BaseChatMessage,
+  ChatGeneration,
+  ChatResult,
+  MessageType,
+} from "../schema/index.js";
+import fetchAdapter from "../util/axios-fetch-adapter.js";
+import type { StreamingAxiosConfiguration } from "../util/axios-types.js";
+import { CallbackManagerForLLMRun } from "../callbacks/manager.js";
+import { BaseLanguageModelCallOptions } from "../base_language/index.js";
+import { getEnvironmentVariable, isNode } from "../util/env.js";
+
+const axios = (
+  "default" in axiosMod ? axiosMod.default : axiosMod
+) as AxiosStatic;
+
+export type WenxinMessageRole = "assistant" | "user";
```

Comment:
Same issue with `enums` unfortunately, they get transpiled into something problematic for other envs